Library/Homebrew/formula_installer.rb

```diff
@@ -1080,6 +1085,8 @@ def install_service
   rescue Exception => e # rubocop:disable Lint/RescueException
     puts e
     ofail "Failed to install service files"
+
+    require "utils/backtrace" if debug?
     odebug e, Utils::Backtrace.clean(e)
```

Comment:
This will probably still break for non-debug cases because the argument to `odebug` is evaluated regardless of debug status.

Reply:
Oh shoot, good point! I'll rework those in a moment.